Fix Cody Web CSS

thenamankumar opened the pull request:
closes: https://linear.app/sourcegraph/issue/SRCH-866/improve-cody-web-theming-for-consistency-with-rest-of-product

This PR updates the Cody Web CSS based on the changes requested by @rrhyne.

Few buttons implemented in Cody doesn't satisfy the design requirements and do not use the css variable so we couldn't update them easily but rather had to override the styles. 

## Test plan

Before: 
![CleanShot 2024-08-08 at 22 53 48@2x](https://github.com/user-attachments/assets/95e10b42-b740-4663-a761-69449ec62296)

After:
![CleanShot 2024-08-08 at 22 54 12@2x](https://github.com/user-attachments/assets/bffbd10a-c87b-4eca-9582-b23eebccb60e)


## Changelog

- Make Cody Web styles more accessible.

rrhyne reviewed (approved):
Love it. Thanks for jumping in and getting this done!

thenamankumar commented on client/web-sveltekit/src/lib/cody/CodySidebarChat.svelte:
```diff
@@ -157,10 +167,71 @@
             box-shadow: unset !important;
         }
+
+        // As of Cody Web 0.4.0 the buttons implemented in Cody does not satisfy
```
As the buttons implemented in Cody does not satisfy the design requirements and it will take longer to design them correctly, then making code changes in Cody, release Cody Web, and then finally update SG. I have been told to override the styles for buttons in SG for the patch release. And the team (me or someone else) will be follow up fixing Cody in next couple of days.

camdencheek commented:
Pulled it down and took a peek. The only thing obvious I noticed was that the `@`-sign color we changed in the sidebar did not get changed in the `/cody/chat` page. No need to block merge here, but noting for followup

![screenshot-2024-08-08_11-36-30@2x](https://github.com/user-attachments/assets/4d0f8bb1-a1c5-4563-853f-98d52df421a0)

thenamankumar commented:
> Pulled it down and took a peek. The only thing obvious I noticed was that the `@`-sign color we changed in the sidebar did not get changed in the `/cody/chat` page. No need to block merge here, but noting for followup
> 
> ![screenshot-2024-08-08_11-36-30@2x](https://private-user-images.githubusercontent.com/12631702/356324100-4d0f8bb1-a1c5-4563-853f-98d52df421a0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMxMzg5NDQsIm5iZiI6MTcyMzEzODY0NCwicGF0aCI6Ii8xMjYzMTcwMi8zNTYzMjQxMDAtNGQwZjhiYjEtYTFjNS00NTYzLTg1M2YtOThkNTJkZjQyMWEwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODA4VDE3MzcyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI3OWI5OWQ5ODZjYzU5ZmYxYTM0YmFkZGEyYTlkOTk2YjU1ODk5ZTY1YzgzMTJhZDdkYTRiMDkzNjQ4YmYwOTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.9qkRxNZ7xRd5x9szAz3I6zJX5202AianEJxHseCtQKY)

updated.
